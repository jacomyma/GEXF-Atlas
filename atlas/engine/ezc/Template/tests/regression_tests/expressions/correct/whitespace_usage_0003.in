0101:[{5 	===5}]
0102:[{5 	=== 	5}]
0103:[{5 	===
5}]
0104:[{5 	===/*c*/5}]
0105:[{5 	===//c
5}]
0106:[{5 	!=5}]
0107:[{5 	!= 	5}]
0108:[{5 	!=
5}]
0109:[{5 	!=/*c*/5}]
0110:[{5 	!=//c
5}]
0111:[{5 	!==5}]
0112:[{5 	!== 	5}]
0113:[{5 	!==
5}]
0114:[{5 	!==/*c*/5}]
0115:[{5 	!==//c
5}]
0116:[{5 	<5}]
0117:[{5 	< 	5}]
0118:[{5 	<
5}]
0119:[{5 	</*c*/5}]
0120:[{5 	<//c
5}]
0121:[{5 	<=5}]
0122:[{5 	<= 	5}]
0123:[{5 	<=
5}]
0124:[{5 	<=/*c*/5}]
0125:[{5 	<=//c
5}]
0126:[{5 	>5}]
0127:[{5 	> 	5}]
0128:[{5 	>
5}]
0129:[{5 	>/*c*/5}]
0130:[{5 	>//c
5}]
0131:[{5 	>=5}]
0132:[{5 	>= 	5}]
0133:[{5 	>=
5}]
0134:[{5 	>=/*c*/5}]
0135:[{5 	>=//c
5}]
0136:[{5 	&&5}]
0137:[{5 	&& 	5}]
0138:[{5 	&&
5}]
0139:[{5 	&&/*c*/5}]
0140:[{5 	&&//c
5}]
0141:[{5 	||5}]
0142:[{5 	|| 	5}]
0143:[{5 	||
5}]
0144:[{5 	||/*c*/5}]
0145:[{5 	||//c
5}]
0146:[{5 	.5}]
0147:[{5 	. 	5}]
0148:[{5 	.
5}]
0149:[{5 	./*c*/5}]
0150:[{5 	.//c
5}]
