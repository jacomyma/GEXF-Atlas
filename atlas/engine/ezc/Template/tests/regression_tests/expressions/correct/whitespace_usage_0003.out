0101:[1]
0102:[1]
0103:[1]
0104:[1]
0105:[1]
0106:[]
0107:[]
0108:[]
0109:[]
0110:[]
0111:[]
0112:[]
0113:[]
0114:[]
0115:[]
0116:[]
0117:[]
0118:[]
0119:[]
0120:[]
0121:[1]
0122:[1]
0123:[1]
0124:[1]
0125:[1]
0126:[]
0127:[]
0128:[]
0129:[]
0130:[]
0131:[1]
0132:[1]
0133:[1]
0134:[1]
0135:[1]
0136:[1]
0137:[1]
0138:[1]
0139:[1]
0140:[1]
0141:[1]
0142:[1]
0143:[1]
0144:[1]
0145:[1]
0146:[55]
0147:[55]
0148:[55]
0149:[55]
0150:[55]
